plugins/parsers/hep/README.md

```diff
@@ -0,0 +1,19 @@
+# HEP
```

Comment:
Due to the large number of integrations using HEP, the name change did not go through as of yet and won't be for the foreseeable future to avoid confusion on an already obscure protocol.